cmd/amp/commands.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"github.com/appcelerator/amp/cli"
+	"github.com/appcelerator/amp/cli/command/version"
+	"github.com/spf13/cobra"
+)
+
+type Opts struct {
```

Comment:
`cmd/amp/commands.go:9:6:warning: exported type Opts should have comment or be unexported (golint)
`